client/src/components/shared/searchbox.js

```diff
@@ -6,15 +6,34 @@ class SearchBox extends Component{
         return(
             <div id="search-box">
                 <ul id='search-options' className="no-padding no-margin text-left list-no-styles">
-                    {this.props.searchResults.filter(word => {
-                        if(word.includes(this.props.value)){
+                    {this.props.searchType === 'members' ? this.props.searchResults.filter(word => {
+                        if(word.firstName.indexOf(this.props.value) > -1 ){
                             return word;
+                        }else{
+                            return ''
                         }
                     }).map(word =>{
                         return (
-                            <Link key={word} to={`/${this.props.searchType}/${word}`}>
-                                <li>{word}</li>
+                            <Link key={word.memberId} to={`/${this.props.searchType}/${word.firstName}+${word.lastName}`}>
+                                <li>
+                                    <img className='memberPhoto' src={'https://theunitedstates.io/images/congress/225x275/' + word.memberId + '.jpg'}  alt=""/>
+                                    <span>{word.firstName}</span>
+                                    <span>{word.lastName} </span></li>
                             </Link>)
+                    }) : this.props.searchResults.filter(word => {
+                            if(word.includes(this.props.value)){
+                                return word;
+                            }else{
+                                return '';
+                            }
+                    }).map(word => {
+                        return(
+                            <Link key={word} to={`/${this.props.searchType}/${word}`}>
```

Comment:
Now that we're using redux, let's put the member or issue name in the redux store as a currentMember or currentIssue variable. That way when we use the search bar in the Navbar, it'll change that variable in the store and cause a refresh of the relevant page. 

Again, we can go through this when we walk through the redux flow.